src/NServiceBus.NHibernate.TransactionalSession/NServiceBus.NHibernate.TransactionalSession.csproj

```diff
@@ -14,7 +14,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="NServiceBus.TransactionalSession" Version="3.2.0" />
+    <PackageReference Include="NServiceBus.TransactionalSession" Version="3.3.0-beta.3" />
```

Comment:
@DavidBoike @bording I think something is off with the code analysis in this repo since this fails the build for all other persisters? (can't ref a prerelease package from a non prerelease component)

Reply:
Already fixed!